tests/HierPathSelect/HierPathSelect.log

```diff
@@ -154,13 +159,15 @@ design: (work@dut2)
       \_ref_obj: (read_buf), parent:read_buf.q
         |vpiName:read_buf
       |vpiActual:
-      \_bit_select: (work@dut2.read_buf.q.q), line:8:17, endln:8:18, parent:read_buf.q
```

Comment:
Huge bug here!

Reply:
Isn't what current one also wrong? Shouldn't `work@dut2.read_buf.q.q` be `work@dut2.read_buf.q`? The variable `q` is `logic` so `q.q` doesn't make any logical sense.

I am gonna revert this specific change and let you deal with it. But there are some issues with parenting and actual_group for `bit_select` & `part_select`. For both the parent should have been a `ref_obj` but it wasn't in this specific case which is what I was attempting to fix.

Reply:
The .q.q is just a printing bug in the vpiFullName when printing, if you traverse the data model, you can find all the nodes required and to the right thing.
With your change, the nodes are gone, that is a real bug.

Is there any way you can unroll all the "other changes" you deem harmless from this PR and only do what it says in the title: Fixing more location information
Hunting for bugs you might have introduced in 400+ log files in not what I had in mind for this week-end....

Reply:
I don't expect you to be spending anytime looking through bugs. That's not your problem. However, please do review the code changes.

I self reviewed my change once again I didn't find anything that wasn't location information specific or _attempt_ to fix bit_select/part_select parenting & actual_group. I have reverted my bad attempt will file bugs against those that I have found.

I have re-based and removed any changes that don't belong with this PR. Feel free to nitpick if you find anything that's not location information specific. Will follow up with an update to logs as well once the build finishes.